cypress/integration/Crosscutting/BackToTopTest.feature

```diff
@@ -0,0 +1,91 @@
+Feature: As a user, I want to be able to use back to top arrow for easy page navigation
+
+    Scenario Outline: Back to top arrow on desktop and tablet on the English site
+        Given user is navigating to "<url>"
+        Given screen breakpoint is set to "<screenBreakpoint>"
+        When user scrolls 600 px down
```

Comment:
this should be 601 px - at 600 it is still not visible

Reply:
pls update this value everywhere

Reply:
done